src/legacy/core_plugins/console/public/np_ready/application/hooks/use_send_current_request_to_es/use_send_current_request_to_es.ts

```diff
@@ -82,5 +93,5 @@ export const useSendCurrentRequestToES = () => {
         });
       }
     }
-  }, [dispatch, settings, history, notifications]);
+  }, [dispatch, settings, history, notifications, trackMetric, METRIC_TYPE.COUNT]);
```

Comment:
My understanding (possibly a misunderstanding) is that this array is intended to define dependencies that may change and, if they do change, a new callback function is generated for the consuming functional component. Is this the case? If so, then I'm not sure `trackMetric` and `METRIC_TYPE.COUNT` belong here because I don't think they're ever going to change.

Reply:
The addition here was to satisfy React's ES lint rules that all dependencies be listed in the dependency array. I expect that none of those should ever change currently :joy: